system/View/Interfaces/ViewDecorator.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\View\Interfaces;
+
+/**
+ * View Decorators are simple classes that are given the
+ * chance to modify the output from the view() calls
+ * prior to it being cached.
+ */
+interface ViewDecorator
```

Comment:
I think that the name of the interface should contain the suffix ``Interface``.

Reply:
I think the current name makes sense in the subdirectory **Interfaces**, but none of our other libraries have a dedicated `Interfaces` namespace so I would be in favor of moving this file up a directory and renaming it as @iRedds suggests.

Reply:
Agree - will make it match the rest of them. I was even the one who initially argued the point that they should be named that way.